src/Data/Audience.elm

```diff
@@ -36,6 +59,56 @@ type alias Audience =
 
 
 
+-- Decoder
+
+
+decodeDataAudience : Decode.Decoder (List Audience)
+decodeDataAudience =
+    Decode.at [ "data" ] (Decode.list decodeAudience)
+
+
+decodeAudience : Decode.Decoder Audience
+decodeAudience =
+    Decode.succeed Audience
+        |> required "id" Decode.int
```

Comment:
Bonus points for using the Elm decoder `|>` operator, not many people use this, but I definitely prefer it! 👏🏻 💯